src/adapters/MeetingsAdapter.js

```diff
@@ -0,0 +1,90 @@
+import {throwError} from 'rxjs';
+
+import WebexAdapter from './WebexAdapter';
+
+/**
+ * This is a base class that defines the interface that maps meetings data.
+ * Developers that want to extend `MeetingsAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class MeetingsAdapter extends WebexAdapter {
+  /**
```

Comment:
Can you move the object definition outside the class, like in [other adapters](https://github.com/webex/components/blob/master/src/adapters/RoomsAdapter.js#L7)?

Reply:
Sure, I copied from https://github.com/webex/components/blob/master/src/adapters/ActivitiesAdapter.js#L14 which has it defined within.